main.tf

```diff
@@ -9,16 +10,23 @@ terraform {
   }
 }
 
-// Get all the Availability Domains for the region
+// Get all the Availability Domains for the region and default backup policies
```

Comment:
You can get many ADs in multi-AD regions. I think this should be renamed to something like ad_list or something similar

Reply:
Opened #77 for this item.